components/list/list-item-generic-layout.js

```diff
@@ -214,7 +214,7 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 
 	connectedCallback() {
 		super.connectedCallback();
-		this.role = this.gridActive ? 'gridrow' : undefined;
```

Comment:
`gridrow` is not even a valid Aria `role`. There are several roles for different purposed, and this is what we want for our usage: `grid` -> `row` -> `gridcell`.